Formula/zeromq.rb

```diff
@@ -11,47 +12,56 @@ class Zeromq < Formula
     sha256 "78c24719c7c8dbba157ca225b41d515190281eb27549caaa60a359ec9f33e3f1" => :yosemite
   end
 
-  head do
-    url "https://github.com/zeromq/libzmq.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
   option :universal
   option "with-libpgm", "Build with PGM extension"
   option "with-norm", "Build with NORM extension"
+  option "with-drafts", "Build and install draft classes and methods"
 
   deprecated_option "with-pgm" => "with-libpgm"
 
+  depends_on "asciidoc" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "pkg-config" => :build
+  depends_on "xmlto" => :build
   depends_on "libpgm" => :optional
   depends_on "libsodium" => :optional
+  depends_on "libtool" => :build
   depends_on "norm" => :optional
 
+  # Avoid known, intermittent "make check" failure
+  # upstream issue "make check error: tests/test_shutdown_stress"
+  # https://github.com/zeromq/libzmq/issues/1509
+  # Autotools dependencies can be removed from stable once this is fixed
+  patch :DATA
```

Comment:
Upstream has agreed to look into my findings:

https://github.com/zeromq/libzmq/issues/1509#issuecomment-260579648